README.md

```diff
@@ -58,24 +58,31 @@ model.then(function(schema) {
 
 Following methods are available on `Schema` instances:
 
-* `castValue(fieldName, value, index, skipConstraints)` - returns a value cast against a named `fieldName`
-* `testValue(fieldName, value, index, skipConstraints)` - returns boolean after a check if value can be casted against a named `fieldName`
 * `castRow(items, failFast = false, skipConstraints = false)` - convert the arguments given to the types of the current schema <sup>1</sup>
-* `fields` - returns an array of objects of the schema's fields
+* `fields` - returns an array of [Field](#field) instances of the schema's fields
 * `foreignKeys` - returns the foreign key property for the schema
-* `getConstraints(fieldName, index = 0)` - return the constraints object for a given `fieldName` <sup>2</sup>
-* `getField(fieldName, index = 0)` - return the field object for `fieldName` <sup>2</sup>
-* `getFieldsByType(typeName)` - return all fields that match the given type
-* `requiredHeaders` - returns headers with the `required` constraint as an array
-* `hasField(fieldName)` - checks if the field exists in the schema. Returns a boolean
+* `getField(fieldName, index = 0)` - returns an instance of [Field](#field) by field name (`fieldName`) <sup>2</sup>
+* `getFieldsByType(typeName)` - returns all fields that match the given type
+* `hasField(fieldName)` - checks if the field exists in the schema by it's name. Returns a boolean
 * `headers` - returns an array of the schema headers
 * `primaryKey` - returns the primary key field for the schema  
```

Comment:
If `schema.primaryKey` could be string or array - use will be get TypeErrors. The reason to wrap descriptor into API us to get some normalization. Because without it use could just use `descriptor.get('primaryKey')`.